hnn_core/dipole.py

```diff
@@ -145,6 +145,101 @@ def simulate_dipole(net, n_trials=1, n_jobs=1):
     return dpl
 
 
+def average_dipoles(dpls):
```

Comment:
Okay, but this might be a bigger issue since `n_trials` would need to be saved with the dipole object to prevent such an error. Right now `Dipole.write()` doesn't do that. Is that what you're suggesting?

Reply:
we could save it too actually. `np.loadtxt` for e.g., allows you to skip few rows in the beginning. You can save this information there. I have seen this kind of thing in other ephys stuff, I think [when saving MNE annotations](https://github.com/mne-tools/mne-python/blob/maint/0.20/mne/annotations.py#L574) ...